Formula/a/advancecomp.rb

```diff
@@ -24,9 +24,8 @@ class Advancecomp < Formula
   uses_from_macos "zlib"
 
   def install
-    system "autoreconf", "-fiv"
-    system "./configure", "--disable-debug", "--disable-dependency-tracking",
-                          "--enable-bzip2", "--prefix=#{prefix}", "--mandir=#{man}"
+    system "autoreconf", "--force", "--install", "--verbose"
```

Comment:
Do we have an audit for this?

Reply:
we dont have audit for this, but I can add one later.

Reply:
yeah, totally.

Reply:
will follow up on this.